sysml/src/training/25. Transitions/Local Clock Example.sysml

```diff
@@ -21,7 +21,7 @@ package 'Local Clock Example' {
 			state waiting;
 			accept request : Request via requestPort
 				then responding;
-			accept at Time::Iso8601DateTime(today + "11:59:00")
+			accept at new Time::Iso8601DateTime(today + "11:59:00")
```

Comment:
Not related to this PR, but should `today` be given a value like "2024-10-25" and then change the string concatenation expression to `today + "T" + "11:59:00"` to create an example that is complete?

Reply:
As noted, this is not really relevant to this PR. It could be considered when the training material is updated. But it is not clear that it is important to the example. Plus, `today` probably shouldn't have a fixed value.

Reply:
OK. The `"T"` is needed in the string concatenation to be a valid `val: Iso8601DateTimeEncoding` actual parameter to `Iso8601DateTime`.